src/app/meetups/view-meetups/meetups-view.component.html

```diff
@@ -20,7 +20,7 @@ <h3 class="margin-lr-3 ellipsis-title">{{meetupDetail.title}}</h3>
   </mat-toolbar>
   <div class="view-container">
     <div class="left-tile">
-      <p i18n><b>Created By:</b> <a [routerLink]="['/users/profile', meetupDetail.createdBy]">{{' ' + meetupDetail.createdBy}}</a></p>
+      <p *ngIf="meetupDetail.createdBy" i18n><b>Created By:</b> <a [routerLink]="['/users/profile', meetupDetail.createdBy]">{{' ' + meetupDetail.createdBy}}</a></p>
```

Comment:
Meetups will always have a `createdBy` when created, so this is unnecessary

Reply:
Thanks Paul. I set condition to show notifications only for user with atleast learner role.

No notification icon for user with no role:
![issue-3324](https://user-images.githubusercontent.com/41846764/55080108-86838600-506b-11e9-9c7e-64bd305eb427.PNG)